.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -49,46 +65,52 @@ This checklist is focused on a classic Jahia repository (module, app)
 - [ ] 🟠 Dependencies listed in packages.json are no more than 2 major versions behind their latest release
 - [ ] 🟠 Linting is executed properly and show no warnings
 - [ ] 🟠 No warning are presents in the browser console when using the app
+- [ ] 🙏 Remove unreachable code
 ### Java
 - [ ] 🔝 Java dependencies are explicitly declared in the module's pom.xml
```

Comment:
This could be automated and integrated in the Maven build with https://maven.apache.org/plugins/maven-dependency-plugin/analyze-only-mojo.html